engine/src/main/java/io/camunda/zeebe/engine/state/migration/to_8_4/corrections/ColumnFamily48Corrector.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.state.migration.to_8_4.corrections;
```

Comment:
❓ 
I guess the packages are wrong then?

Reply:
BTW when can we get rid of them? 8.6 then I guess? Because we expect always minor updates so 8.4 to 8.5.0 (which will have this) and then 8.6.0 it can be removed right?

Reply:
Your assumption is correct, but only if we can guarantee that users can't skip minor updates, including running all migration tasks of that minor version entirely. Currently, we have no guarantee that users adhere to the version requirement, so we can't remove such tasks.